docs/includes/incl-forking-intro.rst

```diff
@@ -0,0 +1,23 @@
+Fork your Aiven service in order to make a copy of the codebase. You can use it to create a development copy of your production environment, set up a snapshot to analyze an issue or test an upgrade, or create an instance in a different cloud/geographical location/under a different plan 
```

Comment:
It's a copy of the database, not the codebase. The original article was trying to make a comparison with codebase forking, since that should be a familiar concept to developers.